README.md

```diff
@@ -1,16 +1,271 @@
+  <a href="https://github.com/folio-org/folio-helm-v2">
+    <img src="https://avatars.githubusercontent.com/u/16495055" alt="FOLIO Helm Charts v2 logo" width="200px" height="200px">
+  </a>
+</p>
+
+<h1 align="center">
+  <div align="center" aria-colspan="0">FOLIO Helm Charts v2</div>
+</h1>
+
+<p align="center">
+  <div align="center">
+    <a href="https://kubernetes.io/releases/download/">
+      <img src="https://img.shields.io/badge/K8S-1.22+-blue?style=for-the-badge" alt="K8S 1.22+" />
+    </a>&nbsp;
+    <a href="https://github.com/folio-org/folio-helm-v2/LICENSE">
+      <img src="https://img.shields.io/badge/License-Apache_2.0-green?style=for-the-badge" alt="License Apache-2.0" />
+    </a>&nbsp;
+    <a href="https://helm.sh/docs/intro/install/">
+      <img src="https://img.shields.io/badge/HELM-3.10+-blue?style=for-the-badge" alt="HELM 3.10+" />
+    </a>&nbsp;
+  </div>
+</p>
 
 ## Introduction
 
-FOLIO Helm charts modules repository.
+This repository contains Helm charts for different modules of the FOLIO (The Future of Libraries is Open) project.
+The FOLIO project aims to reimagine library software through a unique collaboration of libraries, developers, and vendors.
+It looks beyond the current model of an Integrated Library System to a new paradigm, where apps are built on an open platform,
+providing libraries with the ability to innovate and deliver extended services.
+
+In this repository, you will find HELM charts for each FOLIO module.
+
+## Structure
+
+```markdown
+folio-helm-v2/
+├── README.md
+├── example/                        # module chart scaffold
+│   ├── value.yaml                  # library entry point
+│   ├── templates/
+│   │   ├── application.yaml        # main module template model
+│   │   └── NOTES.txt               # chart user instruction model
+│   └── resources                   # resource file examples
+│       ├── ephemeral.properties
+│       └── log4j2.xml
+└── charts/
+    ├── <some-module>/              # FOLIO module HELM chart
+    │   ├── Chart.yaml              # chart information
+    │   ├── values.yaml             # default values for templates
+    │   ├── templates/
+    │   │   ├── application.yaml    # main template
+    │   │   └── NOTES.txt           # chart user instruction
+    │   └── resources/              # chart resource files
+    │       ├── ephemeral.properties
+    │       ├── ..................
+    │       └── log4j2.xml
+    ├── ..................
+    └── folio-common/               # core helpers
+        ├── Chart.yaml
+        ├── values.yaml
+        └── templates/
+            ├── _configmap.tpl
+            ├── _deployment.tpl
+            ├── _helpers.tpl
+            ├── ...............
+            └── _volume.tpl
+```
+
+## Usage
+### Chart usage
+
+The charts in this repository allow users to deploy different modules of the FOLIO project to a Kubernetes cluster using
+the Helm package manager. To do this you can follow the following steps:
```

Comment:
"follow these steps" or "use the following steps"

Reply:
Have done.